app/styles/school-page.less

```diff
@@ -35,4 +35,62 @@
     height: @school-image-map-height;
     background-color: @background-main-color;
   }
+
+  .school-details {
+    section.info-table {
+      .container();
+      margin-top: 30px;
```

Comment:
At some point it would be great to have all of the margin and padding lengths we are using defined as variables. The those would be consistent and easily changeable throughout the service. Optimally they would follow some ratio (the golden ratio for example) to be visually as pleasing as possible. Now it seems we are using at least 10px, 15px, 20px, 30px and 40px. I'll make this into a separate ticket so we can think about it more thoroughly at some point.